common/Reducers.jsm

```diff
@@ -457,6 +462,26 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
       return {...prevState, lastUpdated: INITIAL_STATE.DiscoveryStream.lastUpdated, layout: INITIAL_STATE.DiscoveryStream.layout};
     case at.DISCOVERY_STREAM_FEEDS_UPDATE:
       return {...prevState, feeds: action.data || prevState.feeds};
+    case at.DISCOVERY_STREAM_SPOCS_ENDPOINT:
+      return {
+        ...prevState,
+        spocs: {
+          spocs_endpoint: action.data || prevState.spocs.spocs_endpoint,
```

Comment:
I guess there is intentionally no `lastUpdated` because you want to force the cache to clear?

Reply:
Also why use the previous if none is defined? what if there is intentionally no spocs